service/src/test/java/bio/terra/externalcreds/services/ProviderServiceTest.java

```diff
@@ -139,8 +137,6 @@ private void testWithRevokeResponseCode(HttpStatus httpStatus) {
         providerService.deleteLink(linkedAccount.getUserId(), linkedAccount.getProviderName());
         verify(linkedAccountServiceMock)
             .deleteLinkedAccount(linkedAccount.getUserId(), linkedAccount.getProviderName());
-      } finally {
-        mockServer.stop();
```

Comment:
The `JwtSigningTestUtils` has a teardown method that stops the mockServer: 
```
public void tearDown() { mockServer.stop(); }
```
Should this test class have something similar since you're deleting `mockServer.stop()` line here?

Reply:
the difference is that in `JwtSigningTestUtils` all the tests use the same mock server

Reply:
Okay that makes sense. In that case, should we keep the `mockServer.stop()` line above ^ in `ProviderServiceTest`?

Reply:
Notice I also changed the try on line 104. The mock server is auto closable and will be closed automatically at the end of the try.

Reply:
Thanks for explaining! That makes sense.